x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_logic.ts

```diff
@@ -90,6 +114,15 @@ export const ResultSettingsLogic = kea<MakeLogicType<ResultSettingsValues, Resul
     resetAllFields: () => true,
     updateField: (fieldName, settings) => ({ fieldName, settings }),
     saving: () => true,
+    clearRawSizeForField: (fieldName) => fieldName,
```

Comment:
Even when I'm only passing a single argument, I've been wrapping it like `clearRawSizeForField: (fieldName) => ({fieldName})` because I like the consistency, and also when I realize I need to add another argument later.  This is fine, just wanted to mention my approach.

Reply:
It's definitely the better approach. Kea recommends the same thing:

> Actions are functions that take whatever arguments you choose and return a payload. This payload should always be an object: (amount) => ({ amount }).

Reply:
I was just trying to migrate this, so I moved it over as is.

Reply:
I had to make changes anyway, so I updated this: https://github.com/elastic/kibana/pull/94947/commits/aa858dfea52082d6a879a69b1a76e94eed732b56